apps/hash-ai-worker-ts/src/workflows.ts

```diff
@@ -70,25 +74,283 @@ export const inferEntities = async (params: InferEntitiesCallerParams) => {
   }
 };
 
-type UpdateEntityEmbeddingsParams = {
+type UpdateDataTypeEmbeddingsParams = {
   authentication: {
     actorId: AccountId;
   };
 } & (
   | {
-      entities: Entity[];
+      dataTypes: DataTypeWithMetadata[];
+    }
+  | {
+      filter: Filter;
+    }
+);
+
+export const updateDataTypeEmbeddings = async (
+  params: UpdateDataTypeEmbeddingsParams,
+): Promise<CreateEmbeddingResponse.Usage> => {
+  const temporalAxes = {
```

Comment:
Can we use `currentTimeInstantTemporalAxes` from `@local/hash-isomorphic-utils/graph-queries` instead? Saves having to work out what it is.

Reply:
No, sadly not because this is inside of a workflow and the package depends on the type system package.